src/pages/en/guides/deploy/github.md

```diff
@@ -21,9 +21,17 @@ Astro maintains the official `withastro/action` to deploy your project with very
     If your repository is named `<YOUR_USERNAME>.github.io`, you don’t need to include `base`.
```

Comment:
```suggestion
    If your repository has the same name as your username it will be deployed in `<YOUR_USERNAME>.github.io`, hence, you don’t need to include a `base`.
```

Reply:
@swift502 what do you think about this reword?

Reply:
Shouldn't the same apply for custom domains?

Reply:
@kevinzunigacuellar Ohhhkay, I think I get it. Thanks for being patient with me.

@fredoist is right. You must _not_ include the base if you're using a custom domain, otherwise assets fail to load. I just tested that on my website.

I'd be much more explicit, because it's so convoluted:

> If you're using GitHub's default site url, e.g. `<YOUR_USERNAME>.github.io/my-repo`, set the `base` parameter to your repository’s name starting with a forward slash, for example `/my-repo`. This is so that Astro understands your website's root is "/my-repo", rather than default "/".
> 
> __Don't__ set the `base` parameter if:
> 
> - your repository itself is called `<YOUR_USERNAME>.github.io`
> - you're using a custom domain

---

Combined with the "update your routes" caution box you added, it feels appropriately comprehensive.
Maybe it's a lot of text though. I'll leave it up to you.

Btw:
> If your repository has the same name as your username

I'm not sure that's a good wording because it implies swift502/swift502, which would be the special user repository, not swift502.github.io right?

Reply:
Yes, my mistake here 😅. I decided to keep the part of the original message and add a little more explanation for why and when you need to set a base.

Reply:
LGTM 👍🏼

Reply:
Yeah I like it.